### PR TITLE
updated motivations for GitHub collaboration

### PR DESCRIPTION
**Pull request by JeffOgah**
This pull request addresses [984](https://github.com/Techtonica/curriculum/issues/984)

Motivations for GitHub collaboration lesson updated. 

Included more robust explanation and links with examples.

### CODE REVIEW

File: github-collaboration/github-collaboration.md [hunk 1]
```diff
@@ -13,8 +13,19 @@ About 1 hour
 - Terminal or Command Prompt
 
 ### Motivation
-In the industry, apps are created by teams of developers rather than individual people. Proper use of GitHub for collaboration minimizes code conflicts and helps teams work together.
+- In the industry, apps are created by teams of developers rather than individual people. Proper use of GitHub for collaboration minimizes code conflicts and helps teams work together. 
+- At the heart of collaboration, developers use branches when working on new features or patching a bug. This makes it easier to deal with multiple contexts while developing in a professional manner
```
Comment (alodahl) [inline github-collaboration/github-collaboration.md]:
```suggestion
- At the heart of collaboration, developers use branches when working on new features or patching a bug. This makes it easier to deal with multiple contexts while developing in a professional manner.
```
This looks great so far!  Can you elaborate on what you mean by ' deal with multiple contexts' ?

File: github-collaboration/github-collaboration.md [hunk 2]
```diff
@@ -13,8 +13,19 @@ About 1 hour
...
+- Pull requests (PRs) are how developers include contribution in the intended repository. It lets you tell the repository owners or maintainers about your changes, after which it can be reviewed and included if satisfactory. 
 
+
+**Who uses GitHub:** 
+As of August 2019, more than 40 million people use GitHub. Some popular companies are [Facebook](https://github.com/facebook), [Google](https://github.com/google) and [Microsoft](https://github.com/microsoft). 
+
+**Looking at an example repository**
```
Comment (alodahl) [inline github-collaboration/github-collaboration.md]: love this

Comment (JeffOgah) [reply github-collaboration/github-collaboration.md]: Awesome, thank you

File: github-collaboration/github-collaboration.md (hunk 1 above)
Comment (JeffOgah) [reply github-collaboration/github-collaboration.md]: Oh sorry just saw this, I have explained this here: https://github.com/Techtonica/curriculum/pull/1046 

I deleted my fork when I saw the other PR was merged, I forgot there was an open one here too. Kindly review there while I close this

**Comment by JeffOgah** [comment]
@alodahl Updated PR here: https://github.com/Techtonica/curriculum/pull/1046 
I deleted my fork in error and that affected this PR. Please review on the above

**Pull request closed without merging**